help/styles/styles.css

```diff
@@ -431,4 +431,15 @@ main .section:empty {
 .svg a {
   display: block;
   pointer-events: none;
+}
+
+.button-container a[title="Quick exit"] {
```

Comment:
wouldn't this apply to all `.button-container`? Is it specific to the "Quick exit" buton only?

Reply:
it is specific to only buttons with title "Quick exit"

Reply:
fixed hover color